pass.el

```diff
@@ -413,6 +476,103 @@ If SUBDIR is nil, return the entries of `(password-store-dir)'."
     (remove-text-properties (point-min) (line-end-position)
                             '(display nil))))
 
+(defun pass-view--otp-remaining-secs ()
+  "Return a string with the remaining countdown base 30."
+  (let* ((base 30)
+         (remaining (- base (% (truncate (time-to-seconds (current-time)))
+                               base)))
+         (remaining-str (number-to-string remaining)))
+    (if (< remaining 10)  ;; leftpad-ing
+        (concat "0" remaining-str)
+      remaining-str)))
+
+(defun pass-view--qrcode ()
+  "Open a new buffer that displays a QR Code for the current entry."
+  (let ((entry (pass-view-entry-name))
+        (qrcode-buffer (get-buffer-create "*pass-view-qrcode*")))
+    (with-current-buffer qrcode-buffer
+      (fundamental-mode)
```

Comment:
What major mode would be used if you didn't add `(fundamental-mode)`? Is it really needed?

Reply:
`fundamental-mode` seems to be the default for me

```emacs
(with-current-buffer (get-buffer-create "foobar") major-mode)
```

Reply:
Yes, but I thought maybe I was missing something.

Reply:
I switch to `fundamental-mode` because the way I see it, it is possible that the buffer `*pass-view-qrcode*` already exists and it's currently in `image-mode` (say, they user didn't close the previous QR code that they opened), so in order to edit it and add the SVG code, I change *back* it to `fundamental-mode`.

Reply:
Maybe a one-line comment would be better than an `unless` call then.